components/controls/km-questions.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <div  v-if="answersInitialized" >
+    <div class="mb-3" v-for="question in questions" :key="question.key">           
+        <div v-if="!question.questions?.length">
+            <km-question :question="question" v-model="answers[question.key]" @update:modelValue="onAnswer"></km-question>
+        </div>
+        <div v-if="question.questions?.length">
+            <!-- <CCard>
```

Comment:
![Screenshot 2024-03-25 at 12 54 56 PM](https://github.com/scbd/online-reporting-tool/assets/13351610/c3703dab-ddf9-46da-b39d-6a7ff6ec9209)

Comments in the template break git hubs colouring.  making harder to read.  for future.  I will ensure same.  U have  a lot of nuxt and vue coming for review soon.